shared/src/business/entities/cases/Case.test.js

```diff
@@ -4138,4 +4190,18 @@ describe('Case entity', () => {
       expect(hasPartyWithPaperService).toBeFalsy();
     });
   });
+
+  // TODO: enable for issue 7080
```

Comment:
I think this test is for issue #7166 and got mixed up with #7080 - if so, I think we can enable this now.

Reply:
No, I'm wrong about this. Sorry, disregard!